docs/configure-rails/guardrail-catalog/self-check.md

```
@@ -19,6 +19,22 @@ This category of rails relies on prompting the LLM to perform various tasks like
 You should only use the example self-check prompts as a starting point. For production use cases, you should perform additional evaluations and customizations.
 ```
 
+### Reasoning Models as Self-Check LLMs
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)